examples/apps/cli/CMakeLists.txt

```diff
@@ -26,41 +26,23 @@
 #  POSSIBILITY OF SUCH DAMAGE.
 #
 
-add_executable(ot-cli-ftd
-    main.c
-)
+add_executable(ot-cli-ftd main.c)
 
-add_executable(ot-cli-mtd
-    main.c
-)
+add_executable(ot-cli-mtd main.c)
 
-set(COMMON_INCLUDES
-    ${OT_PUBLIC_INCLUDES}
-    ${OT_PRIVATE_INCLUDES}
-    ${PROJECT_SOURCE_DIR}/examples/platforms
-    ${PROJECT_SOURCE_DIR}/src/core
+set(COMMON_INCLUDES ${OT_PUBLIC_INCLUDES} ${OT_PRIVATE_INCLUDES} ${PROJECT_SOURCE_DIR}/examples/platforms
+                    ${PROJECT_SOURCE_DIR}/src/core
 )
 
 target_include_directories(ot-cli-ftd PRIVATE ${COMMON_INCLUDES})
 target_include_directories(ot-cli-mtd PRIVATE ${COMMON_INCLUDES})
 
-target_link_libraries(ot-cli-ftd PRIVATE
-    openthread-cli-ftd
-    ${OT_PLATFORM_LIB}
-    openthread-ftd
-    ${OT_PLATFORM_LIB}
-    mbedcrypto
-    ot-config
+target_link_libraries(
+    ot-cli-ftd PRIVATE openthread-cli-ftd ${OT_PLATFORM_LIB} openthread-ftd ${OT_PLATFORM_LIB} mbedcrypto ot-config
```

Comment:
I think the original format is more readable. can we customize the configuration file of cmake-format to use the original format?